arangod/Replication2/StateMachines/BlackHole/Monostate.h

```diff
@@ -31,9 +33,9 @@ struct Monostate {
       // We could also check whether we got null here.
       return arangodb::inspection::Status();
     } else {
-      return f.apply(arangodb::inspection::Null{});
+      return f.apply(arangodb::velocypack::Slice::emptyObjectSlice());
```

Comment:
In this case the comment is now outdated. When we represent it as empty object we could also do this:
```
  return f.object(p).fields();
```
Then we wouldn't need the specialization for the loading case. By default `ignoreUnknownFields` is false, so we would actually produce an error in case we got some unexpected fields.